patch/attiny841.yaml

```diff
@@ -0,0 +1,66 @@
+_svd: ../svd/attiny841.svd
+
+_include:
+  - "common/port.yaml"
+  - "common/spi.yaml"
+  - "common/usart.yaml"
+  - "common/wdt.yaml"
+
+  - "timer/attiny841.yaml"
+
+AC:
+  _modify:
+    ACSR?A:
+      access: read-write
+  ACSR?A:
+    _modify:
+      ACIS?:
+        description: "Analog Comparator Interrupt Mode Select"
+      ACO:
+        access: read-only
+    ACIS?:
+      _replace_enum:
+        ON_TOGGLE: [0, "Interrupt on Toggle"]
+        # Leaving [1, 'Reserved'] out
+        ON_FALLING_EDGE: [2, "Interrupt on Falling Edge"]
+        ON_RISING_EDGE: [3, "Interrupt on Rising Edge"]
```

Comment:
You can make the common patch work here as well by adding the following changes:

```diff
 AC:
   _modify:
-    ACSR:
+    ACSR,ACSR?A:
       access: read-write
-  ACSR:
+  ACSR,ACSR?A:
     _modify:
-      ACIS:
+      ACIS,ACIS?:
         description: "Analog Comparator Interrupt Mode Select"
       ACO:
         access: read-only
-    ACIS:
+    ACIS,ACIS?:
       _replace_enum:
         ON_TOGGLE: [0, "Interrupt on Toggle"]
         # Leaving [1, 'Reserved'] out

```